google-signin-aware.html

```diff
@@ -460,13 +460,13 @@
             // Let the current user listener trigger the changes.
           },
           function error(error) {
-            if ("Access denied." == error.reason) {
-              // Access denied is not an error, user hit cancel
-              return;
-            } else {
-              console.error(error);
+            // Access denied is not an error, user hit cancel
+            if ("Access denied." !== error.reason) {
+              this.signinAwares.forEach(function(awareInstance) {
+                awareInstance.fire('google-signin-aware-error', error);
```

Comment:
Not dodgy, that's the pattern we use with this element. One thing I would have done differently is that instead of firing the event right here, I'd instead notify `signin-aware`, and let it fire the event. That way, signin-aware will be aware of the error, if we want to do something else in the error handler in the future.
```javascript
awareInstance.errorNotify(error);

errorNotify: function(error) {
  this.fire('google-signin-aware-error', error);
}
```